src/components/shared/Dialog/dialogTypes/ConfirmLogout.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Button from '@material-ui/core/Button';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import Translate from '../../../Translate/Translate.react';
+
+const ConfirmLogoutDialog = props => {
+  const { title, content, handleConfirmLogout, handleClose } = props;
+  const [loading, handleLoading] = useState(false);
+
+  const handleConfirm = () => {
```

Comment:
Try to make this dialog generic, as confirming an action has use-cases at many other places as well. Try to pass the  action handler, title, body text as props.